packages/rs-sdk/src/mock/wallet/minimal.rs

```diff
@@ -0,0 +1,20 @@
+use std::fmt::Display;
```

Comment:
Is this by mistake?

Reply:
yes, good point, missed this.

Will remove.